### DISCUSSION TITLE
Configure and track channels changes

### DISCUSSION THREAD

**Discussion by ADhiab92** [saleor/saleor#11478] [Deployment] [answered]
I have 6 warehouses in different places or districts, that means I have 6 channels in saleor, and each warehouse there is an employee that has access to change the quantities of the inventory through saleor, but i don't want him to change the quantities of every warehouse but limit the changes so he can change only the warehouse that he works at, and will not be able to change the other channels (other warehouses). Is there a way to do so if not can we track who changed the quantity of a specific items?

**Comment by timuric** [comment, marked as answer]
@ADhiab92 There is an RFC for channel permission that should solve your case https://github.com/saleor/saleor/pull/10423
